src/soundcloud.js

```diff
@@ -0,0 +1,31 @@
+import { URL } from 'url';
+
+export const shouldTransform = string =>
+  new URL(string).host === 'soundcloud.com';
+
+const getSoundcloudIFrameSrc = string =>
+  `https://w.soundcloud.com/player` +
```

Comment:
Don't like the `+` sign tbh.
We're using [template literals](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals), but we're still using the `+` sign.

This feels a bit grim to me actually.

But I like the cleanliness of the separate lines too. 😕